### PR TITLE
[PSDK] SYSTEM_POWER_STATUS: Rename Reserved1 to SystemStatusFlag

### PR DESCRIPTION
**Pull request by SergeGautherie**
as on Win10.

Cherry pick
https://source.winehq.org/git/wine.git/commit/7c5ea436af8a6b5a3d2b095bd4697f10b6f78acc

### CODE REVIEW

File: sdk/include/psdk/winbase.h [hunk 1]
```diff
@@ -1151,7 +1151,11 @@ typedef struct _SYSTEM_POWER_STATUS {
 	BYTE ACLineStatus;
 	BYTE BatteryFlag;
 	BYTE BatteryLifePercent;
+#if _WIN32_WINNT < _WIN32_WINNT_WIN10
```
Comment (ThFabba) [inline sdk/include/psdk/winbase.h]: Do we use `Reserved1` anywhere? If not, we don't need this ifdef. It's unconditionally called `SystemStatusFlag` in PSDK.

Comment (SergeGautherie) [reply sdk/include/psdk/winbase.h]: ReactOS itself does not use this field.
Nonetheless, shouldn't PSDK and test be both consistent with S2003 target? As in usable separately, especially such a compile-time test.

Comment (ThFabba) [reply sdk/include/psdk/winbase.h]: No, our headers should match the latest official ones.

Comment (SergeGautherie) [reply sdk/include/psdk/winbase.h]: Ah. Done.

**Pull request merged**